src/components/database/create-database/CreateDatabase.jsx

```diff
@@ -117,15 +127,28 @@ const CreateDatabase = (props) => {
           </Form.Item>
         </Form>
         <Alert message={alertMsg}
-            description=" "
-            type="info"
-            showIcon />
+          description=" "
+          type="info"
+          showIcon />
         <Form>
-        <p style={{ marginBottom: 0, marginTop: "2%", fontWeight:"bold" }}>Alias</p>
-        <label style={{ fontSize: 12 }}>Alias name is used in your frontend queries to identify your database</label>
+          <p style={{ marginBottom: 0, marginTop: "2%", fontWeight: "bold" }}>Alias</p>
+          <label style={{ fontSize: 12 }}>Alias name is used in your frontend queries to identify your database</label>
           <Form.Item>
             {getFieldDecorator('alias', {
-              rules: [{ required: true, message: 'Please input an alias for your database' }],
+              rules: [{
+                validator: (_, value, cb) => {
+                  if (!value) {
+                    cb("Please input an alias for your database")
+                    return
+                  }
+                  allAlias.map(data => {
```

Comment:
use `array.some` method here to figure out if the alias name is already taken and then use that information to call the `cb` array with error or without error